span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanNormalizer.java

```diff
@@ -63,6 +64,11 @@
   private final JaegerResourceNormalizer resourceNormalizer = new JaegerResourceNormalizer();
   private final TenantIdHandler tenantIdHandler;
 
+  // measure the span's start time and its processing time
```

Comment:
```suggestion
  // measure the difference between span's start time and its processing time
```

Reply:
Done via UI to handle in a single commit.